src/platform/plugins/shared/discover/public/application/main/components/tabs_view/tabs_view.tsx

```diff
@@ -33,6 +34,21 @@ export const TabsView = (props: SingleTabViewProps) => {
   const { getPreviewData } = usePreviewData(props.runtimeStateManager);
   const hideTabsBar = useInternalStateSelector(selectIsTabsBarHidden);
 
+  const tabsEbtManager = useMemo(
+    () => services.ebtManager.createScopedEBTManager(),
+    [services.ebtManager]
+  );
```

Comment:
```suggestion
  const scopedEbtManager = useCurrentTabRuntimeState(
    props.runtimeStateManager,
    (state) => state.scopedEbtManager$
  );
```
We should avoid creating a separate EBT manager, and instead pull the one for the current tab from the runtime state.